src/mvs_eland/E0_evaluation.py

```diff
@@ -154,6 +154,21 @@ def evaluate_dict(dict_values, results_main, results_meta):
             E2.lcoe_assets(dict_values[group][asset], group)
             store_result_matrix(dict_values[KPI], dict_values[group][asset])
 
+    # detect excessive excess generation in any bus
+    for bus_label, bus_df in dict_values[OPTIMIZED_FLOWS].items():
+        # disregard excess sinks
+        cols = [col for col in bus_df.columns if "excess" not in col]
+        df = pd.DataFrame(bus_df[cols].sum(), columns=["sum"])
+        # get total in- and outflow of bus
+        total_inflow_bus = df[df > 0].dropna().sum().values[0]
+        total_outflow_bus = df[df < 0].dropna().sum().values[0]
```

Comment:
This is a very neat way to do it :)

If we run into errors we will need to check if I we did all the prefixes correctly while defining `OPTIMIZED_FLOWS` .